backend/app/converters/lib/marcxml_auth_agent_base_map.rb

```diff
@@ -48,7 +48,6 @@ def agent_family_base(import_events)
   def shared_subrecord_map(import_events)
     h = {
       'parent::record/leader' => agent_record_control_map,
-      "parent::record/controlfield[@tag='001'][not(following-sibling::controlfield[@tag='003']/text()='DLC' and following-sibling::datafield[@tag='010'])]" => agent_record_identifiers_base_map("//record/controlfield[@tag='001']"),
```

Comment:
@quoideneuf - Is there a reason for this deletion?

Reply:
@lorawoodford - I wasn't sure if it was doing anything, but maybe it is?

Reply:
Yea, it seems curious to me that tests don't fail on this, but I do recall there was a lot of convoluted logic on when/how/where from record identifiers should be imported.

Reply:
@lorawoodford I put it back.